docs/secure-connections/sauce-connect/advanced/specifications.md

```diff
@@ -0,0 +1,141 @@
+---
+id: specifications
+title: Sauce Connect Proxy Specifications
+sidebar_label: Specifications
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+If you're a security or network administrator, you may find it useful to review the Sauce Connect Proxy security specifications to better evaluate its security.
+If you use Sauce Connect Proxy as part of your CI/CD pipeline, you may find it useful to review timeout values to better configure CI/CD success/failure parameters.
```

Comment:
@waggledans I thought I remembered you had made a comment in this section -- just wanted to make sure it was captured!

Reply:
yeah, I think now it mentions CI/CD. Good enough